packages/localize-tools/testdata/build-runtime-xliff-ph/goldens/tsout/empty.txt

```diff
@@ -0,0 +1 @@
+git won't track an empty directory
```

Comment:
(is this necessary? looks like there are files in this dir)

Reply:
Done